data/json/mapgen/lab/lab_surface/lab_surface_big_z1.json

```diff
@@ -0,0 +1,247 @@
+        "#...............................................__________________________________________________________________,_,__#",
+        "#...............................................__________________________________________________________________,__,_#",
+        "#...............................................____________________________,,____,,_____,,_____,,____,,____,,____,,__,#",
+        "#...............................................____________________________,,____,,_____,,_____,,____,,____,,____,_,__#",
+        "#...............................................____________________________,,____,,_____,,_____,,____,,____,,____,__,_#",
+        "#...............................................____________________________,,____,,_____,,_____,,____,,____,,____,,__,#",
+        "#...............................................____________________________,,____,,_____,,_____,,____,,____,,____,_,__#",
+        "#...............................................____________________________,,____,,_____,,_____,,____,,____,,____,__,_#",
+        "#...............................................____________________________,,____,,_____,,_____,,____,,____,,____,___,#",
+        "#...............................................________________________,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,#",
+        "#...............................................____________________________________________________________,..........#",
+        "#...............................................____________________________________________________________,..........#",
+        "#..............................................]____________________________________________________________,..........#",
+        "#.............................................]]____________________________________________________________,..........#",
+        "#............................................]]_____________________________________________________________,..........#",
+        "#...........................................]]______________________________________________________________,..........#",
+        "#..........................................]]_______________________________________________________________,..........#",
+        "#..........]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]],,,,,,,_________________]]]]]]]]]]]]]]]]]]]]]]]]]]],,,,,,,,]]..........#",
+        "#..........]|  #########################  |]***],,,,,,,_________________]***]|  ###################__,__,__|]..........#",
+        "#..........]   :     #&T#T&#           :   ]***]_________________,,,,,,,]***]   #|||||||||||#&T#T&#,__,__,_ ]..........#",
+        "#..........]   :     #  #  #           :   ]***]_________________,,,,,,,]***]   #|eeee|b <>|#  #  #_,__,__, ]..........#",
+        "#..........]|  #     # S#S #           #  |]***]________________________]***]|  #|eeee|b <>|# S# S#__,__,__|]..........#",
```

Comment:
In the bathrooms at the loading dock, the eastern one has the sink in front of the door:

```
#&T#T&#
#  #  #
# S# S#
#+###+#
```
should probably be
```
#&T#T&#
#  #  #
# S#S #
#+###+#
```